src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Task is class for instantiating objects tasks
+ * Tasks are used in TaskList
+ *
+ * @wangwaynesg (Wang Wayne)
+ */
+
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task (String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[\u2713]" : "[\u2718]");
```

Comment:
Might be good to specify in a way that these are HTML codes for ticks and crosses,, i would consider it as borderline magic number?

Reply:
Good idea! I will implement that!